aospy/test/data/objects/examples.py

```diff
@@ -47,6 +47,13 @@ def total_precipitation(convection_rain, condensation_rain):
     models=[example_model]
 )
 
+precip_largescale = Var(
```

Comment:
Why is this new `Var` object needed?

Reply:
I just needed to test a `Var` object with `def_time=False`.

Reply:
OK.  Let's rename it to make that explicit; otherwise it's confusing why there are two nearly identical objects.  E.g. `var_not_time_defined = Var(name='var_no_time_def', def_time=False)`.  You can leave `alt_names` and `description` unspecified.